Dockerfile.assisted_installer_agent

```diff
@@ -0,0 +1,3 @@
+FROM registry.fedoraproject.org/fedora-minimal:latest
```

Comment:
Please discuss with @YuviGold on what should be the base container

Reply:
OK

Reply:
@ronniel1 : @YuviGold thinks fedora-minimal is OK, so I will continue with it.